### DISCUSSION TITLE
Asking For Help: How to make AutoDJ skip song when a DJ connects? + How to increase General Output Volume?

### DISCUSSION THREAD

**Discussion by unudelaradio** [AzuraCast/AzuraCast#6988] [Q&A]
Hi guys! I'm asking once again for your help on 2 different topics:

**Topic 1**
Is there some workaround to make AutoDJ skip the current song when a Streamer/DJ connects? This is a known issue: since few versions of AzuraCast, after the Streamer/DJ disconnects, a fragment of the last song (the song before Live started) is still playing, for a few seconds, together with the track that should normally start. This transitions worked fine before. The AutoDJ was fading out before switching to Live and the song was skipped, i guess. So when the DJ was stopping the broadcast, the transition was perfect, starting with the next song from queue.

**Topic 2**
How can i _correctly_ increase my General Output Volume? I'm using Stereo Tool with a nice preset, i found it on the internet. But i want to set the output volume a little bit louder. Since i don't know how to edit Stereo Tool presets... please tell me if my current method is correct: i simply pasted the next line in the _4th_ box of Liquidsoap Config:

`radio = amplify(override="liq_amplify", 1.1, radio)` 

Playing with values like `1.` then `1.1` and `1.2` i discovered that the output volume is changing and i think Stereo Tool preset remains active and untouched. Am i doing things right? Does this line of code may affect the Stereo Tool preset or other Azuracast Settings?

Also i'm trying to understand how the values work. Is `1.1` a correct value? I also tried with `1.2` but the output result was already too loud for my desire. If my method is correct, how would these values will be translated in _dB_ format? Because i want to avoid red peaks and sound distorsion.

**Comment by Heimatradio** [comment]
for editing the preset use the Standalone version of StereoTool, you can download it at [https://www.thimeo.com/stereo-tool/download/](url)

**Comment by JeffersonStarseed** [comment]
Yes, even with a single StereoTool license (that you've installed on your Azuracast), you are allowed to have a standalone version of it on your personal or work computer for the purposes of working with and tweaking the settings, creating/editing your presets, and checking-out new presets.  However, I found out that my standalone install didn't take the same license key, so I still get the occassionally StereoTool advertisement-drop.  Not a big deal, since its only for tweaking/experimentation on my personal computer and not for my "live" environment on my radio server.

But I get what you're asking, and I was wondering the same thing on your second Topic.  Is the "amplify" parameter in LiquidSoap ...before StereoTool's input? ...or is it after the StereoTool output and (essentially) the final parameter before going out to the listener?
